java/org/contikios/cooja/contikimote/ContikiMote.java

```diff
@@ -57,7 +57,7 @@
  *
  * @author      Fredrik Osterlind
  */
-public class ContikiMote extends AbstractWakeupMote implements Mote {
+public class ContikiMote extends AbstractWakeupMote<ContikiMoteType,SectionMoteMemory> implements Mote {
```

Comment:
Spaces after the commas like in AbstractWakeupMote?

Reply:
Fixed.